src/BenchmarkDotNet/Exporters/Json/JsonExporterBase.cs

```diff
@@ -21,51 +22,69 @@ protected JsonExporterBase(bool indentJson = false, bool excludeMeasurements = f
         }
 
         public override void ExportToLog(Summary summary, ILogger logger)
+        {
+            JsonSerializer.CurrentJsonSerializerStrategy.Indent = IndentJson;
+            logger.WriteLine(JsonSerializer.SerializeObject(CreateLog(summary)));
+        }
+
+        protected virtual IReadOnlyDictionary<string, object> CreateLog(Summary summary)
```

Comment:
nit: I am not a big fan of the `CreateLog` name. I would rather call this method `GetDataToSerialize`